testdriver/src/testdriver/gen_fake_parliamentary.bal

```diff
@@ -237,6 +237,8 @@ function genFakeParliamentary() {
                 {
                     "party_code": "Bar",
                     "party_name": "Bar bar",
+                    "vote_count": 0,
```

Comment:
Does the logic break without this? I think we should handle the scenario where this maybe unavailable too? Or we have to confirm that this will always be available?